controllers/vehicle/curt.go

```diff
@@ -42,8 +42,14 @@ func CurtLookup(w http.ResponseWriter, r *http.Request, enc encoding.Encoder, dt
 	if hdstr == "true" {
 		heavyduty = true
 	}
+	// determine if you are going to get customer prices for each part
+	var getCustomerPrices bool
+	custPricingStr := r.URL.Query().Get("customerPrices")
+	if custPricingStr == "true" || custPricingStr == "True" {
```

Comment:
I did this because its more efficient and this way I didn't have to include the strings package, but can do. Changing it to:

if strings.ToLower(custPricingStr) == "true" {